assets/csidriveroperators/ovirt/07_deployment.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ovirt-csi-driver-operator
+  namespace: openshift-cluster-csi-drivers
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: ovirt-csi-driver-operator
+  template:
+    metadata:
+      labels:
+        name: ovirt-csi-driver-operator
+    spec:
+      serviceAccountName: ovirt-csi-driver-operator
```

Comment:
07_deployment.yaml was just updated, please pull it again.

Reply:
Updated. And added CredentialRequest sync for oVirt to CSO.